tests/coverage-run-rustdoc/doctest.rs

```diff
@@ -1,4 +1,5 @@
 //@ aux-build:doctest_crate.rs
+//@ ignore-backends: gcc
```

Comment:
Nit: Since we're now skipping the whole suite in the Dockerfile, this test doesn't need an explicit ignore.

Reply:
Good catch, thanks!